doc/site/modules/core/list.markdown

```diff
@@ -110,3 +110,40 @@ backwards from the end of the list where `-1` is the last element.
     System.print(list) //> [a, new, c]
 
 It is a runtime error if the index is not an integer or is out of bounds.
+
+### **sort**()
+
+Sorts the list in place.
```

Comment:
drive-by documentation nit: "... using the < operator."

In the core library, it looks like only the Num type has this operator so far.

(Also, since you're using quicksort, maybe say that it's not a stable sort?)